config/settings.yml

```diff
@@ -1,3 +1,30 @@
+# the first endpoint_type listed must be for an local "online"
+# endpoint, and the default values listed for it will be used
+# in combination with the defined storage_roots to seed the db
+# with endpoints.
+endpoint_types:
+  local_online_nfs:
+    endpoint_class: online
+    endpoint_defaults:
+      endpoint_node: 'localhost'
+      recovery_cost: 1
+
 moab:
-  storage_roots: 'spec/fixtures'
+  # this is a list of named storage roots.  the names are used to seed the endpoints table.
+  storage_roots:
+    # name: '/location/on/disk'  # e.g.
+    fixtures: 'spec/fixtures'
   storage_trunk: 'moab_storage_root'
+
+# the first preservation policy listed will be considered the "default"
+# policy, to be used when a policy is otherwise left unspecified
+preservation_policies:
+  default:
+    archive_ttl: 604800 # _ttl values are in seconds
+    fixity_ttl: 604800
```

Comment:
this is probably nitpicking .... but later in the code you put 604_800 because I'm guessing rubocop like that.

Reply:
i actually just wasn't sure whether you could do the underscore separator thing in yml files, and never bothered to try it out.  but happy to give it a shot if we'd prefer to be more consistent about that.

Reply:
FWIW, just tested this out in `irb`, yml does support this:

```2.4.1 :003 > YAML.load("600_000")
 => 600000```

I'd prefer consistency, but in whatever direction -- if we want to disable that rubocop rule, I'd be fine with that, too.

Reply:
will fix in the yml.  or, fine to disable the rubocop rule, if people would prefer that.  no strong feelings either way from me.  i might have a slight preference for getting rid of the rubocop rule, but fine with whatever.

Reply:
I think it would be good to indicate that 604_800 is a week's worth of seconds (if that's the correct answer).